vnext/react-native.config.js

```diff
@@ -2,16 +2,77 @@
 require('source-map-support').install();
 
 const cli = require('@react-native-windows/cli');
+const { execSync } = require('child_process');
```

Comment:
shouldn't all this logic be going into the CLI instead of in the RNW project's config?

Reply:
Yes, I think this would be better in the CLI. This also lets it be typescript.

Reply:
Admittedly this one is also kind of funky because we have the PowerShell script sitting in vnext, and the PowerShell script is publicly documented.